tests/fsharp/single-test.fs

```diff
@@ -139,7 +139,7 @@ let generateProjectArtifacts (pc:ProjectConfiguration) outputType (targetFramewo
         let template = @"<Project Sdk='Microsoft.NET.Sdk'>
 
   <PropertyGroup>
-    <DisableAutoImports>true</DisableAutoImports>
+    <NoWarn>$(NoWarn);NU1604;NU1103</NoWarn>
```

Comment:
Might also do `<PackageReference Remove="<mibc_packages>"` or something.

Reply:
Okay so this I think I hacked around, tests pass now